lib/rules/number-max-precision/README.md

```diff
@@ -37,3 +37,46 @@ a { top: 3.24px; }
 ```css
 @media (min-width: 3.23em) {}
 ```
+## Optional secondary options
```

Comment:
Please add one blank line above the heading.

(Note: Let's see if remark can lint for this).